### ISSUE TITLE
voiceover screenreader bug

### ISSUE THREAD

**Issue by kaveinthran** [Automattic/pocket-casts-ios#1320]
### Description

Hi, I am using the latest iOS and here are some of the issues that I found when using it with voiceover,

1. on the upnext "queue" dialog, voiceover lags when scrolling, I have about 1189 episodes and it's hard to scroll past, the percentage verticle bar  that should be there to scroll a screen is not shown to voiceover 
2. the queue dialog have select episode and reorder button can be added to roter action to minimise swipe and increase screenreader efficiency 
3.  the navigation in episodes list of a podcast  need to be fixed. At the moment, when I swipe right to explore the previous episodes, it is good, but when  swipe left it got stuck after 4 or five episode, I need to pull the screen down with three finger swipe down to scroll, and it doesn't automatically scrol. It also happens when I search for episodes in the episode list. the swipe left is not infinite.
4. the shownote for an episode isn't read automatically to screenreader when cursor is focused on an episode. user need to enter into an episode and go to the bottom of the screen to read the shownote which is alot of cognitive load.
5. the filter tab on create a new filter dialog isn't screenreader efficient, the podcast list is hard to scroll, no index navigation or search, if a filter list is created, there's no way to identify which podcast is added to the list inside the list, when selecting a podcast to be added, there's no confirmation whether the podcast is selected.
6. 

### Step-by-step reproduction instructions

all of above issues are found when using voiceover on the IPhone

### Did you search for existing bug reports?

- [X] I have searched for existing bug reports.
